app/views/layouts/_navbar.html.erb

```diff
@@ -1,9 +1,18 @@
 <div class="navbar-wagon">
   <!-- Logo -->
+  <div class="search-box">
```

Comment:
Maybe and an icons for the search. You can use <i class="fas fa-search"></i> from font-awesine